docs/tutorials/integrations/kserve/kserve-custommetrics-one.md

```diff
@@ -0,0 +1,117 @@
+This tutorial shows how easy it is validate SLOs for a single model in [KServe](https://kserve.github.io/website/0.10/) when fetching metrics from a metrics database like Prometheus. We show this using the `sklearn-iris` model used for a [first `InferenceService`](https://kserve.github.io/website/master/get_started/first_isvc/) in the KServe documentation.
+
+???+ "Before you begin"
+    1. Try [your first experiment](../../../getting-started/your-first-experiment.md). Understand the main [concepts](../../../getting-started/concepts.md) behind Iter8 experiments.
+    2. Ensure that you have the [kubectl](https://kubernetes.io/docs/reference/kubectl/) CLI.
+    3. Have access to a cluster running [KServe](https://kserve.github.io/website). You can create a [KServe Quickstart](https://kserve.github.io/website/0.10/get_started/#before-you-begin) environment as follows:
+    ```shell
+    curl -s "https://raw.githubusercontent.com/kserve/kserve/release-0.10/hack/quick_install.sh" | bash
+    ```
+    These instructions can be used to [install Prometheus](https://github.com/kserve/kserve/tree/master/docs/samples/metrics-and-monitoring#install-prometheus).
+
+
+## Experiment Setup
+
+Deploy the model and generate load against it. We follow the instructions for the [KServe first InferenceService](https://kserve.github.io/website/master/get_started/first_isvc) to deploy the model.
+
+### Create InferenceService for Initial Model
+
+```shell
+kubectl apply -f - <<EOF
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "sklearn-iris"
+spec:
+  predictor:
+    model:
+      modelFormat:
+        name: sklearn
+      storageUri: "gs://kfserving-examples/models/sklearn/1.0/model"
+EOF
+```
+
+### Generate Load
+
+Port forward requests to the cluster:
+
+```shell
+INGRESS_GATEWAY=$(kubectl get svc --namespace istio-system --selector="app=istio-ingressgateway" --output jsonpath='{.items[0].metadata.name}')
+kubectl port-forward --namespace istio-system svc/$INGRESS_GATEWAY 8080:80
+```
+
+Finally, request predictions. Here we make a request about once a second.
+
+```shell
+SERVICE_HOSTNAME="sklearn-iris.default.example.com"
+# kubectl get inferenceservice sklearn-iris -o jsonpath='{.status.url}' | cut -d "/" -f 3
+
+
+cat <<EOF > "./iris-input.json"
+{
+  "instances": [
+    [6.8,  2.8,  4.8,  1.4],
+    [6.0,  3.4,  4.5,  1.6]
+  ]
+}
+EOF
+
+while true; do 
+  curl -H "Host: ${SERVICE_HOSTNAME}" \
+    http://localhost:8080/v1/models/sklearn-iris:predict \
+    -d @./iris-input.json
+  sleep 1
+done
+```
+
+## Launch Iter8 Experiment
+
+Launch the Iter8 experiment inside the Kubernetes cluster:
+
+```shell
+iter8 k launch \
+--set "tasks={ready,custommetrics,assess}" \
+--set ready.isvc=sklearn-iris \
+--set ready.timeout=180s \
+--set custommetrics.templates.kserve-prometheus="https://gist.githubusercontent.com/kalantar/adc6c9b0efe483c00b8f0c20605ac36c/raw/c4562e87b7ac0652b0e46f8f494d024307bff7a1/kserve-prometheus.tpl" \
```

Comment:
perhaps this template should be in [samples]( https://github.com/iter8-tools/docs/tree/main/samples) instead of a gist.

Reply:
Good for now ... we should eventually eliminate URLs for these types of task inputs eventually.